src/SmartComponents/modules/__tests__/reducer-test.js

```diff
@@ -12,23 +12,33 @@ describe('compare reducer', () => {
                 factFilter: '',
                 filterDropdownOpened: false,
                 selectedSystemIds: [],
-                stateFilter: 'all'
+                stateFilter: 'all',
+                filteredData: {},
```

Comment:
For clarity, I think maybe we change `compare` to something like `fullResults` or `fullCompareData` and then this to `filteredCompareData`.

Reply:
Sorry, I know this comment is in a test, but I don't mean it for the test, but generally across the app.